outlines/welcome.md

```diff
@@ -0,0 +1,13 @@
+1. Welcome to the Meteor guide
+  1. This is a set of articles outlining opinions on best-practice application development in the Meteor framework
+  2. Nothing here is prescriptive (i.e. you can use the platform in many other ways), but this is how we think apps are best developed.
+  3. Hard APIs can be found at [docs] and community packages at [atmosphere].
+2. What is Meteor?
+  1. A curated set of JS client + server code for building web and mobile applications
+  2. A build tool for building JS bundles for deployment
+3. Meteor concepts
+  - Should we move the concepts from the docs here?
```

Comment:
I'd add the principles of Meteor or similar: http://docs.meteor.com/#/full/sevenprinciples

All of the rest of the "concepts" in the docs are already represented in the Guide (Security, etc) so we should just remove them from the docs once we publish and link to the guide instead.